xbmc/addons/kodi-addon-dev-kit/include/kodi/kodi_game_dll.h

```diff
@@ -46,7 +46,7 @@ const char* GetGameAPIVersion(void);
  *
  * \return Must be GAME_MIN_API_VERSION_STRING
  */
-const char* GetMininumGameAPIVersion(void);
+const char* GetMinimumGameAPIVersion(void);
```

Comment:
This breaks ABI for some game add-ons. Not against this, but it needs its own PR so we can track down all the places this is used.